db/schema.rb

```diff
@@ -123,8 +123,8 @@
   create_table "distributors_shipping_methods", :force => true do |t|
     t.integer  "distributor_id"
     t.integer  "shipping_method_id"
-    t.datetime "created_at",         :null => false
-    t.datetime "updated_at",         :null => false
+    t.datetime "created_at"
+    t.datetime "updated_at"
```

Comment:
We shouldn't run `rake db:migrate` every time in development because different versions of Rails could generate different schema. I suggest to just run `rake db:schema:load` to make sure your branch doesn't touch the schema if not needed.

If your branch adds new migrations just run `rake db:migrate:up VERSION={MIGRATION_ID}` and commit the changes that your migration introduce.

Does this make sense? The risk here is to mix commits and change the DB schema when it's not related with your PR.